examples/haas.cfg.dev-no-hardware

```diff
@@ -0,0 +1,96 @@
+# This is an example configuration file for HaaS. The ``haas`` command will
```

Comment:
Please either keep the word "example" in this file name or remove it from haas.cfg.dev-example; there's no reason for the inconsistency.